.github/workflows/helm.yaml

```diff
@@ -0,0 +1,33 @@
+name: helm sanity
+
+on:
+  pull_request:
+    branches: [ '**' ]
+  push:
+    branches: [ 'master' ]
+  workflow_dispatch:
+    branches: [ '**' ]
+
+jobs:
+  helm_sanity:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+
+      - uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+        with:
+          project_id: ${{ secrets.GCP_PROJECT_ID }}
+          service_account_key: ${{ secrets.GCP_SA_KEY }}
+          export_default_credentials: true
+
+      - uses: azure/setup-helm@v1
+        with:
+          version: v3.2.4
+
+      - name: run tests
+        run: |
+          gcloud container clusters get-credentials ${{ secrets.CLUSTER_NAME }} --region ${{ secrets.REGION }}
```

Comment:
is there any way to run minikube on git action? instead of maintaining a real cluster

Reply:
why minikube?
There is no maintenance here

Reply:
Well we do need a cluster that will be available 24/7 only for a small script now and then..

Reply:
@val06 what do you think?

Reply:
bummer
I'll switch to minikube